app/Http/Controllers/V2/Dashboard/GetProjectsController.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Http\Controllers\V2\Dashboard;
+
+use App\Helpers\TerrafundDashboardQueryHelper;
+use App\Http\Controllers\Controller;
+use App\Http\Resources\V2\Dashboard\GetProjectsResource;
+use Illuminate\Http\Request;
+
+class GetProjectsController extends Controller
+{
+    public function __invoke(Request $request): GetProjectsResource
+    {
+        $projects = TerrafundDashboardQueryHelper::buildQueryFromRequest($request)
+            ->whereNotNull('long')
+            ->whereNotNull('lat')
+            ->select('uuid', 'long', 'lat', 'name')
+            ->get();
+
+        return new GetProjectsResource([
+            'data' => $projects,
+        ]);
```

Comment:
please drop the resource in cases like this `return response()->json(['data' => $projects])`